docs/data/joy/components/linear-progress/LinearProgressCountUp.js

```diff
@@ -25,8 +25,8 @@ export default function LinearProgressCountUp() {
       thickness={24}
       value={Number(value)}
       sx={{
-        '--LinearProgress-radius': '20px',
-        '--LinearProgress-thickness': '24px',
+        [linearProgressCssVars.radius]: '20px',
```

Comment:
This is what I was referring to in question two, of my PR. 

Should it be left as the string values? personally I feel like this should be the case...